packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -5,8 +5,8 @@
 %global gem_name hammer_cli
 
 %global release 1
-%global prereleasesource pre.develop
-%global prerelease %{?prereleasesource:.}%{?prereleasesource}
+#%global prereleasesource pre.develop
+#%global prerelease %{?prereleasesource:.}%{?prereleasesource}
```

Comment:
IIRC -- you should just delete this or you have to do `%%` to disable the macro, comments dont actually disable it

Reply:
Yay rpm....  fixeded